autogpt_platform/frontend/src/components/ui/BoringAvatarWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import Avatar from "boring-avatars";
```

Comment:
This only exists as otherwise there is a naming clash when using it in avatar.tsx